source/extensions/http/header_validators/envoy_default/header_validator.h

```diff
@@ -19,38 +22,138 @@ class HeaderValidator : public ::Envoy::Http::HeaderValidator {
           config,
       ::Envoy::Http::Protocol protocol, StreamInfo::StreamInfo& stream_info);
 
-  HeaderEntryValidationResult
-  validateRequestHeaderEntry(const ::Envoy::Http::HeaderString& key,
-                             const ::Envoy::Http::HeaderString& value) override;
+  using HeaderValueValidationResult = RejectResult;
+  /*
+   * Validate the :method pseudo header, honoring the restrict_http_methods configuration option.
+   */
+  virtual HeaderValueValidationResult
+  validateMethodHeader(const ::Envoy::Http::HeaderString& value);
+
+  /*
+   * Validate the :status response pseudo header based on the range of valid response statuses.
+   * TODO: add RFC reference
```

Comment:
AFIK TODOs should have names or issues attached.
This is especially important for TODOs which aren't under the header validators directory like the codec TODOs, so we can grep for anything outstanding associated with the issue.

Reply:
The RFC reference was moved into the implementation.

Reply:
Ok but there's many other TODOs in this PR.
Also this was resolved and I don't see a change.  I think I'll 
/wait 
on the assumption there's something to push and I'll take a look once that's done :-)